packages/reports/tests/acceptance/multi-datasource-test.js

```diff
@@ -22,40 +22,6 @@ module('Acceptance | multi-datasource report builder', function (hooks) {
     config.navi.FEATURES.enableVerticalCollectionTableIterator = false;
   });
 
-  test('dimension filter with metric operations', async function (assert) {
```

Comment:
Was this failing? We don't want to delete it since it's the only test for dimensions filters with metric operations